recipes/leparagliding/LICENSE

```diff
@@ -1,674 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Please don't edit other recipe files, those will be taken care of automatically.